scripts/template/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "reactotron-template",
+  "version": "0.0.1",
+  "description": "A Reactotron plugin",
+  "author": "Infinite Red",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/infinitered/reactotron/issues"
+  },
+  "homepage": "https://github.com/infinitered/reactotron/tree/master/lib/reactotron-template",
+  "repository": "https://github.com/infinitered/reactotron/tree/master/lib/reactotron-template",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "main": "dist/index.js",
+  "module": "dist/index.esm.js",
+  "types": "dist/types/src/index.d.ts",
+  "react-native": "src/index.ts",
+  "exports": {
+    "default": "./dist/index.js",
+    "import": "./dist/index.esm.js",
+    "types": "./dist/types/src/index.d.ts",
+    "react-native": "./src/index.ts"
+  },
+  "scripts": {
+    "test": "jest --passWithNoTests",
+    "test:watch": "jest --watch --notify",
+    "format": "prettier '*.{js,ts,tsx,json,md,css,yml}|**/*.{js,ts,tsx,json,md,css,yml}'",
+    "format:check": "yarn format --check",
+    "format:write": "yarn format --write",
+    "prebuild": "yarn clean",
+    "build": "yarn tsc && yarn compile",
+    "prebuild:dev": "yarn clean",
+    "build:dev": "yarn tsc && yarn compile:dev",
+    "clean": "rimraf ./dist",
+    "lint": "eslint 'src/**/**.{ts,tsx}'",
+    "compile": "NODE_ENV=production rollup -c rollup.config.ts",
+    "compile:dev": "NODE_ENV=development rollup -c rollup.config.ts",
+    "tsc": "tsc",
+    "typecheck": "tsc",
+    "ci:test": "yarn test --runInBand"
+  },
+  "peerDependencies": {
+    "reactotron-core-client": "*"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
+    "@babel/preset-typescript": "^7.23.2",
+    "@types/jest": "^29.5.7",
+    "@types/node": "^18.18.8",
+    "@typescript-eslint/eslint-plugin": "^6.7.5",
+    "@typescript-eslint/parser": "^6.7.5",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^29.7.0",
+    "eslint": "^8.53.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.0",
+    "eslint-plugin-n": "^16.2.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-standard": "^5.0.0",
+    "jest": "^29.7.0",
+    "prettier": "^3.0.3",
+    "react-native-mmkv": "^2.10.2",
+    "reactotron-core-client": "workspace:*",
+    "rollup": "^1.1.2",
+    "rollup-plugin-babel": "^4.4.0",
+    "rollup-plugin-babel-minify": "^7.0.0",
+    "rollup-plugin-filesize": "^6.0.1",
+    "rollup-plugin-node-resolve": "^4.0.0",
+    "rollup-plugin-resolve": "^0.0.1-predev.1",
+    "testdouble": "^3.20.0",
```

Comment:
We'll want to remove `testdouble`. I forgot to remove this in another PR but we want to move away from this